reposerver/repository/repository.go

```diff
@@ -1023,9 +1023,9 @@ func runHelmBuild(appPath string, h helm.Helm) error {
 	manifestGenerateLock.Lock(appPath)
 	defer manifestGenerateLock.Unlock(appPath)
 
-	// the `helm dependency build` is potentially time consuming 1~2 seconds
-	// marker file is used to check if command already run to avoid running it again unnecessary
-	// file is removed when repository re-initialized (e.g. when another commit is processed)
+	// the `helm dependency build` is potentially a time-consuming 1~2 seconds,
+	// a marker file is used to check if command already run to avoid running it again unnecessarily
```

Comment:
This might not be related to the changes in this PR, but maybe reword this?

Reply:
How would you reword this?

Reply:
```suggestion
	// a marker file is used to check if command already ran to avoid running it again unnecessarily
```